docs/troubleshooting/import-backups.md

```diff
@@ -0,0 +1,56 @@
+---
+slug: import-backups/
+id: import-backups
+title: How to import a backup without being signed in
+sidebar_label: Import backups
+description: How to import a backup in the Standard Notes web and desktop app without being signed in.
+keywords:
+  - standard notes
+  - docs
+  - notes app
+  - end-to-end encryption
+  - web
+  - desktop
+  - import backups
+  - backups
+  - import
+  - data
+  - account
+image: /img/logo.png
+hide_title: false
+hide_table_of_contents: false
+---
+
+## Introduction
+
+We are aware of issues with importing backups. If you are signed in, some tags are duplicated and are not properly assigned to notes and a significant number of items (notes and tags) are not imported. We are working on a fix. As a temporary workaround, you can import the backup while signed out of the app before signing in to the account. Then, when you sign in, choose to merge local data (this is an option that is on by default).
```

Comment:
I believe this issue only applies if you are importing a backup belonging to another account, where that account is still registered. So there is no global import issue, just this particular case. It would help to specify this.

Reply:
I didn't realize that the issue is specific because Jaspal's original commit suggested that you can use an existing account that's cleared out: https://github.com/standardnotes/docs/pull/26/commits/c887050e177cad0816e6d5b4f34fba107d0dd5a7#diff-1fef0bd7d07b38a1e9cb95fa0889e04e2a86e57dc5af37eefea98e77eb69b6a5R38

But anyways, I made it clearer here: https://github.com/standardnotes/docs/commit/082ee41494a393c269a3bc05c0ba160e016bcc86.